extensions/linear/src/components/IssueActions/IssueActions.tsx

```diff
@@ -477,7 +477,7 @@ export default function IssueActions({
 
         <Action
           title="Delete Issue"
-          shortcut={{ modifiers: ["cmd", "shift"], key: "d" }}
+          shortcut={{ modifiers: ["ctrl"], key: "x" }}
```

Comment:
```suggestion
          shortcut={Keyboard.Shortcut.Common.Remove}
```
You can use above where we store the global hotkeys, remember to import it too